Sources/SwiftKuery/Column.swift

```diff
@@ -59,18 +77,30 @@ public class Column: Field, IndexColumn {
         return _table
     }
     
-    /// Initialize an instance of Column.
-    ///
+    // MARK: Column Initializer
+    
+    /**
+     The initializer for the `Column` class. This creates an instance of `Column` using the provided parameters.
+     Name must be provided, but all other fields will default to either nil or false if not given.
+     ### Usage Example: ###
+     In this example, an instance of the `Column` class is created to match The person_id column of an SQL table.
```

Comment:
How about "In this example, an instance of the `Column` class is created to match the `person_id` column of an SQL table. The `person_id` is set as the primary key, which must be unique and cannot be null so the parameters `unique` and `notNull` are set to true."